tools/format_python_tools.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import fnmatch
+import os
+import sys
+
+from yapf.yapflib.yapf_api import FormatFile
+
+EXCLUDE_LIST = ['pyformat']
+
+
+def collectFiles():
+  """Collect all Python files in the tools directory.
+
+  Returns: A collection of python files in the tools directory excluding
+    any directories in the EXCLUDE_LIST constant.
+  """
+  # TODO: Add ability to collect a specific file or files.
+  matches = []
+  for root, dirnames, filenames in os.walk('.'):
+    dirnames[:] = [d for d in dirnames if d not in EXCLUDE_LIST]
+    for filename in fnmatch.filter(filenames, '*.py'):
+      matches.append(os.path.join(root, filename))
+  return matches
+
+
+def validateFormat(fix=False):
+  """Check the format of python files in the tools directory.
+
+    Arguments:
```

Comment:
Nit: remove two space here and in line below to match other Python arg styles. It would be nice if the tool could enforce this one as well :)